lighthouse.ts

```diff
@@ -21,5 +21,5 @@ export const runLighthouseReport = async (url: string) => {
     throw new Error(`Lighthouse report failed for: ${url}`);
   }
 
-  return stdout;
+  return String(stdout).replace(/\r\n/g, '\n');
```

Comment:
It turns out `typeof stdout` was `object`, not `string`, it looks like it was a string-like thing that is not actually a string. `stdout.replace(...)` didn't do anything since it wasn't a string, so I had to convert it to string first

Reply:
Weird!?! Nice job figuring that out.

Reply:
What's our confidence level that this will work on PCs as well as Mac? My vague understanding is that the OSs use different line endings, and so universal line endings have to be some combination of white space characters that both understand. I'm guessing you tested on a Mac already; is there any chance this fix will break the tool for PC users? If we're not sure, do you have access to a PC for testing this? (I do not)

Reply:
Yeah I can test it out

Reply:
Perfect. Whenever that's confirmed, go ahead and merge/publish this fix. No need for me to re-review this.

Reply:
![image](https://user-images.githubusercontent.com/13206945/97486917-5cfea080-1919-11eb-8170-4acfc2f529c2.png)

It's working 🎉